models/issue.go

```diff
@@ -1734,7 +1735,7 @@ func GetUserIssueStats(opts UserIssueStatsOptions) (*IssueStats, error) {
 	}
 
 	switch opts.FilterMode {
-	case FilterModeAll:
+	case FilterModeAll, FilterModeYourRepositories:
```

Comment:
I think this is a break change, isn't that?

Reply:
I don't think so. The current behavior for the "Your repositories" was actually not limiting it to the user's repositories and rather get all issues made by the user.